test/e2e/images_test.go

```diff
@@ -105,6 +105,16 @@ var _ = Describe("Podman images", func() {
 		Expect(len(session.OutputToStringArray())).To(Equal(1))
 	})
 
+	It("podman images filter reference", func() {
+		if podmanTest.RemoteTest {
+			Skip("Does not work on remote client")
+		}
+		result := podmanTest.Podman([]string{"images", "-q", "-f", "reference=docker.io*"})
```

Comment:
Lets add a couple of others.

Look for only alpine, make sure you get one
Look for a*pine a make sure you get one.
and look for bogus.  and make sure you get none.

Reply:
fixed :raising_hand_woman: